PWGHF/DataModel/CandidateReconstructionTables.h

```diff
@@ -35,6 +35,7 @@
 
 #include "PWGHF/Core/SelectorCuts.h"
 #include "PWGHF/Utils/utilsPid.h"
+#include "PWGHF/Utils/utilsTrkCandHf.h"
```

Comment:
This should only be included in files that use `countOnesInBinary`.

Reply:
Done